src/graphql/get-users.js

```diff
@@ -65,3 +65,43 @@ export const GET_USER_WATCHLIST = gql`
     }
   }
 `;
+
+export const GET_USERS_WATCHLIST = gql`
```

Comment:
I think this is great but, it gets all user watchlists, not just the _**current user's**_ like the query above this.